logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -154,6 +154,11 @@ interface ConversationRepository {
     suspend fun deleteMembersFromEvent(userIDList: List<UserId>, conversationID: ConversationId): Either<CoreFailure, Unit>
     suspend fun observeOneToOneConversationWithOtherUser(otherUserId: UserId): Flow<Either<CoreFailure, Conversation>>
 
+    suspend fun getOneOnOneConversationsWithOtherUser(
+        otherUserId: UserId,
+        protocol: Conversation.Protocol
+    ): Either<StorageFailure, List<ConversationId>>
```

Comment:
thought: I see it allows specifying the protocol for the 1:1. So it does not get the "active" 1:1 conversation.

I'm continuing this on the next comment.